docs/tutorial/sandbox.md

```diff
@@ -0,0 +1,136 @@
+# Process Sandboxing
+
+One key security feature in Chromium is that processes can executed within a sandbox.
+The sandbox limits the harm that untrusted code can cause by limiting access to most
+system resources — sandboxed processes can only freely use CPU cycles and memory.
+In order to perform operations requiring additional privilege, sandboxed processes
+use dedicated communication channels to delegate tasks to more privileged processes.
+
+In Chromium, sandboxing is mainly applied to renderer processes. In addition, certain
+utility processes are also sandboxed, including the audio service, the GPU service,
+and the storage service. On Windows, the `chrome://sandbox` page will show you the exact
```

Comment:
Is their no macOS equivalent? If not I'm not sure it's worth mentioning as it might just confuse folks as to what they can't see, the majority of devs I assume are on macOS or linux 🤔

Reply:
According to Google Analytics, about half of our traffic is actually from Windows. That's also approximately in line with the rate of tickets we receive on Discord.

As for alternatives, there's no mention of them in the [Chromium doc I found](https://www.chromium.org/Home/chromium-security/articles/chrome-sandbox-diagnostics-for-windows).

Reply:
Sandboxing mechanisms are vastly divergent on different OSes, it makes sense that this wouldn't be easily replicable on a different OS.

Perhaps we should avoid mentioning this altogether as it's not really information that is useful to the average developer.

Reply:
Done in b75691f